platform-sdk/platform-apps/tests/MigrationTestingTool/src/main/java/com/swirlds/demo/migration/MigrationTestingToolState.java

```diff
@@ -245,7 +247,7 @@ public void init(
         }
 
         // FUTURE WORK: this needs to be updated once we create the next saved state for the MigrationTestingTool
-        if (previousSoftwareVersion != SoftwareVersion.NO_VERSION) {
+        if (!Objects.equals(previousSoftwareVersion, SOFTWARE_VERSION)) {
             logger.error(
                     MARKER,
                     "previousSoftwareVersion was {} when expecting SoftwareVersion.NO_VERSION",
```

Comment:
this message should be updated, as `NO_VERSION` is not what's being expected

Reply:
whoops!  will be fixed.